pyERGM/utils.py

```diff
@@ -1005,3 +1005,19 @@ def sample_from_independent_probabilities_matrix(probability_matrix, sample_size
             sample[i, j, :] = np.random.binomial(1, probability_matrix[i, j], size=sample_size)
 
     return sample
+
+
+# TODO: njit?
+def split_network_for_bootstrapping(net_size: int, first_part_size: int, splitting_method: str = 'uniform') -> tuple[
+    np.ndarray[int], np.ndarray[int]]:
+    if splitting_method == 'uniform':
+        first_part_indices = np.random.choice(net_size, size=first_part_size, replace=False).reshape(
+            (first_part_size, 1)).astype(int)
+    else:
+        raise ValueError(f"splitting method {splitting_method} not supported")
+
+    second_part_indices = np.array(
```

Comment:
I wonder if this could be sped up (we know for-loops are slow)

e.g. - if np.shuffle is faster, we can shuffle and split

Reply:
but anyway other split methods will probably be slower, so I guess there's no need to waste much time on this

Reply:
Actually the shuffle and cut indeed sounds like a much better solution